### ISSUE TITLE
главная.иконка "@"-переход не на ту страницу

### ISSUE THREAD

**Issue by ttanchik11** [ttanchik11/test.mail.kino#3]
главная страница вторая полоса иконок

"как воспроизвести"
1- открыть главную страницу сайта https://kino.mail.ru
2- При наведение на иконку "@" Всплывает краткое писание иконки "на mail.ru" 
![Снимок экрана 2020-08-02 в 11 09 33](https://user-images.githubusercontent.com/69103554/89118836-b0e02180-d4b1-11ea-8ccc-3ffd703800ef.png)

3-нажать на иконку -открывается главная страницу проекта https://kino.mail.ru
[видео экрана кнопка @1.mov.zip](https://github.com/ttanchik11/test.mail.kino/files/5011856/%401.mov.zip)

Ожидаемый результат:
при нажатие на "@" Открытие главное  страница mail.ru
![Снимок экрана 2020-08-02 в 11 13 54](https://user-images.githubusercontent.com/69103554/89118845-c5241e80-d4b1-11ea-948d-b9558ab4ef74.png)

**Comment by Svetlana678** [comment]
Не баг. Система перехода отрабатывает корректно. Пр нажатии на @ - видим "на  mail.ru", наживаем и переходим на mail.ru
При наведении курсора на слово "КИНО" - видим "На Главную кино", нажимаем и переходим на главную страницу  проекта кино. По оформлению нужно назначить правопреемника и выставить красную этикетку баг (ошибка) при заведении.

**Comment by ttanchik11** [comment]
В сафари моей версии именно так как я написала.
Возможно в других браузерах иначе.
Данные баги говорят о неадаптивности именно к сафари на макинтоше

Отправлено с iPhone



Отправлено с iPhone

**Comment by Svetlana678** [comment]
Тогда в описании к ошибке, нужно было указать, на каком именно браузере воспроизводится ошибка.

**Comment by ttanchik11** [comment]
Все указано. В том числе в отчете по тесту в заглавных критериях

Отправлено с iPhone

**Comment by Svetlana678** [comment]
Нет, нужно оказывать это в описании к ошибке, нам об этом рассказывали на лекциях, шаги воспроизведения должны быть составлены так,чтобы, любой мог по ним воспроизвести ошибку. И если ошибка воспроизводится только в одном определенном браузере,это нужно обязательно указывать.
